### PR TITLE
Add In the News section

### LINKED ISSUE #4: Add an 'In the news' section

### PR DESCRIPTION
**Pull request by kevincianfarini**
Closes #4

### CODE REVIEW

File: docs/news.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ---
 # Feel free to add content and custom Front Matter to this file.
 # To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
-layout: page
+layout: page 
+title: In the News
+permalink: /news/
 ---
+
+{% include news/news.html %}
```
Comment (kevincianfarini) [inline docs/news.md:9]: I would have loved to write this as content in markdown, but unfortunately the minima jekyll theme is too restrictive to allow linking to an external site as a 'post'. So instead I ripped the HTML and edited it manually and just embed that here. It works quite well.

**Comment by kevincianfarini** [comment]
image

**Comment by eshilts** [comment]
It would be good to get a sense of how we want to add new links in the future. Is it by copypastaing the html and changing the link?

**Comment by kevincianfarini** [comment]
Yeah for now it is.

**Pull request merged**